lib/Common/Memory/RecyclerWriteBarrierManager.h

```diff
@@ -63,12 +76,20 @@ class X64WriteBarrierCardTableManager
     // Get the card table for the 64 bit address space
     BYTE * GetAddressOfCardTable() { return _cardTable; }
 
-#if DBG
+#if ENABLE_DEBUG_CONFIG_OPTIONS
     void AssertWriteToAddress(_In_ void* address)
     {
         Assert(_cardTable);
         Assert(committedSections.Test(GetSectionIndex(address)));
     }
+    BOOLEAN IsCardTableCommited(_In_ uintptr_t index)
```

Comment:
need to cast then, prefast warns perf impact, it's debug code though, let me change it

---
In reply to: [87501538](https://github.com/Microsoft/ChakraCore/pull/1920#discussion_r87501538) [](ancestors = 87501538)